packages/type/scss/_reset.scss

```diff
@@ -7,6 +7,7 @@
 
 @import '@carbon/layout/scss/convert';
 @import 'font-family';
+@import './styles.scss';
```

Comment:
I'd go with `@import 'styles';` for consistency. Also, if reset imports font-family and styles, should `index.scss` be updated to this, to prevent unnecessary imports?

```scss
@import 'reset';
@import 'scale';
```

With `type.scss` getting similar changes?